lib/src/collision/broadphase/default_broadphase_buffer.dart

```diff
@@ -110,18 +111,18 @@ class DefaultBroadPhaseBuffer implements TreeCallback, BroadPhase {
     // Send the pairs back to the client.
     int i = 0;
     while (i < _pairCount) {
-      Pair primaryPair = _pairBuffer[i];
-      Object userDataA = _tree.getUserData(primaryPair.proxyIdA);
-      Object userDataB = _tree.getUserData(primaryPair.proxyIdB);
+      int primaryPair = _pairBuffer[i];
+      // print("Pairs => ${primaryPair >> 32} ${primaryPair & _mask}");
```

Comment:
I think this was left here accidentally?

Reply:
Yes in fact, now fixed 👍